torchao/csrc/nms.cpp

```diff
@@ -0,0 +1,7 @@
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/library.h>
+#include <torch/types.h>
+
+TORCH_LIBRARY_FRAGMENT(torchao, m) {
```

Comment:
did you write this or was it codegened?

Reply:
This is not codegenned. We could codegen it, but in general codegen adds more complexity. What were you thinking for the API?